src/App.tsx

```diff
@@ -1,42 +1,37 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './types/Tab';
+import { findCorrectTab } from './utils/helpers';
 
-export const tabs = [
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const currentTab = findCorrectTab(tabs, selectedTabId);
+
+  const handleButtonClick = (tab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (tab: Tab) => {
```

Use more specific name